packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -94,13 +96,15 @@
     "@itwin/presentation-components": "^4.1.0",
     "@itwin/presentation-frontend": "^4.1.0",
     "@itwin/webgl-compatibility": "^4.1.0",
+    "@tanstack/query-core": "^4.36.1",
```

Comment:
This is used as a non-dev dependency.

Reply:
Rush/pnpm refuses to install this transient dependency. So the builds fail.

Reply:
Then move it to non-dev dependency list.

Reply:
This is a quirk with pnpm and workspaces. If you are consuming this package normally it should install on its own.

Reply:
Quirk or not, you are referencing it directly in code. Add it as a dependency.

Reply:
Why, when react-query already has it as a dependency?
![image](https://github.com/iTwin/viewer-components-react/assets/26608199/bab008d6-be18-4971-89e4-ddfd31c8cf2b)

Reply:
So why do you even need to add the `@tanstack/query-core` into any kind of dependency list if "react-query already has it as a dependency"?

If you would be referencing `query-core` directly in tests I would say keep it this way. But now you are referencing it directly in production code. I don't get it why you should make it a dev dependency.

I think if you wan to keep it this way then for consistency you should move `classnames` (transitive dependency from `@itwin/itwinui-react`) as well.

Reply:
I think I get you're point. Actually, I didn't realize that react-query was reexporting the packages. Sadly, I still need this as a dev dependency, since it is not hoisting it correctly.

Reply:
I am not saying that you don't need it. I am saying that you are using it as a dependency, not as a devDevependency.

Reply:
Yup I get it. Nice catch. I was too focused on the pnpm issue at hand that I tunneled visioned.

Reply:
Changed the import.